airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/staging/StagingConsumerFactory.java

```diff
@@ -157,16 +151,16 @@ private RecordWriter recordWriterFunction(final JdbcDatabase database,
       final WriteConfig writeConfig = pairToWriteConfig.get(pair);
       final String schemaName = writeConfig.getOutputSchemaName();
       final String tableName = writeConfig.getOutputTableName();
-      final String path = namingResolver.getStagingPath(schemaName, tableName, CURRENT_SYNC_PATH);
+      final String path = namingResolver.getStagingPath(RANDOM_CONNECTION_ID, schemaName, tableName, CURRENT_SYNC_PATH);
```

Comment:
why do we need a random ID here?

Reply:
I wish we could put the actual connection_id of the sync, that way a load from staging to warehouse could leverage data that was uploaded to stage in a previous attempt but failed to load to the warehouse for some reasons (interrupted?) instead.

But we don't have that ID available for now, so I'm just using a random one as a placeholder for the moment. This would avoid mixing data in the staging area between different syncs (especially if they manipulate streams with similar names)